code/headers/flame_sensor.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <hwlib.hpp>
+
+namespace r2d2::flame_sensor {
+    class flame_sensor_c {
+    private:
+        hwlib::adc *leds[5];
```

Comment:
Personally i would like to see std::array

Reply:
changed https://github.com/R2D2-2019/flame_sensor/pull/1/commits/e71eeeda7b190e0b21024b520ce1826bcaa8f57e